addons/web_editor/static/src/js/wysiwyg/widgets/link_dialog.js

```diff
@@ -12,6 +12,7 @@ export class LinkDialog extends Link {
         focusField: { type: String, optional: true },
         close: { type: Function },
         onSave: { type: Function },
+        hasButtonDialog: { type:Boolean, optional: true},
```

Comment:
Can't we remove this too then and always hardcode the title and button text as mentioned in my previous comment?

Reply:
Actually, as per the spec, for /link and for /button we have to give different titles and confirm-text and both of these models use the same templates and functions. so to differentiate that I had used this one. Tried earlier but did not find any other way to differentiate them.

Reply:
I had a quick chat with Luc and he said it's ok to simplify and put just "Insert a Link / Button" as the title and "Insert" as the button label.